networkit/community.pyx

```diff
@@ -418,6 +418,34 @@ cdef class PLM(CommunityDetector):
 	def prolong(Graph Gcoarse, Partition zetaCoarse, Graph Gfine, vector[node] nodeToMetaNode):
 		return Partition().setThis(PLM_prolong(Gcoarse._this, zetaCoarse._this, Gfine._this, nodeToMetaNode))
 
+cdef extern from "<networkit/community/ParallelLeiden.hpp>":
+
+	cdef cppclass _ParallelLeiden "NetworKit::ParallelLeiden"(_CommunityDetectionAlgorithm):
+		_ParallelLeiden(_Graph _G) except +
+		_ParallelLeiden(_Graph _G, int iterations, bool_t randomize, double gamma) except +
+
+cdef class ParallelLeiden(CommunityDetector):
```

Comment:
Please add a simple Python test (mainly to test the Python interface).

Reply:
Is there a similar one for plm?
Truth is, I haven't touched python at all so all I'm really doing there is copying plm and eyeballing which changes are needed lol.

Reply:
Yes, here is an example: https://github.com/networkit/networkit/blob/master/networkit/test/test_algorithms.py#L307